package.json

```diff
@@ -8,8 +8,9 @@
     "clean": "rm -rf packages/wonder-blocks-*/dist && rm -rf packages/wonder-blocks-*/node_modules",
     "lint": "eslint --config ./eslint/eslintrc packages",
     "lint:watch": "esw --watch --config ./eslint/eslintrc packages",
-    "build:all": "webpack --display-modules && node utils/package-flow-types.js",
-    "watch:all": "webpack -w",
+    "build:cjs": "webpack --display-modules && node utils/package-flow-types.js",
```

Comment:
You could also keep a `build:all` which then runs both `build:cjs` and `build:es6`.